bee-node/docker/Dockerfile

```diff
@@ -0,0 +1,42 @@
+############################
+# Build
+############################
+FROM rust:1-buster as build
+
+ARG WITH_DASHBOARD=false
+
+LABEL org.label-schema.description="Bee node software for the IOTA mainnet."
```

Comment:
Not sure about this description, aren't we going to use the same file for shimmer but also mainnets/testnets ?